src/sss_client/pam_sss.c

```diff
@@ -995,7 +995,7 @@ static int eval_response(pam_handle_t *pamh, size_t buflen, uint8_t *buf,
                     break;
                 }
 
-                if (type == SSS_PAM_CERT_INFO && pi->cert_user == '\0') {
+                if (type == SSS_PAM_CERT_INFO && *pi->cert_user == '\0') {
```

Comment:
really a nitpick from my side, but I'd prefer to have it as *(pi->cert_user) == '\0' ...